distribution/lib/Standard/Base/0.0.0-dev/src/Data/List.enso

```diff
@@ -317,12 +318,12 @@ type List
              example_init = Examples.list.init
     init : List ! Empty_Error
     init self =
-        init' x y = case y of
+        init_fn x y = case y of
             Nil -> Nil
-            Cons a b -> Cons x (init' a b)
+            Cons a b -> Cons x (init_fn a b)
```

Comment:
oof, glad this is changed

Should we even allow identifiers like that?? Possibly a question to @kazcw

Reply:
not one for this PR but I will replace any I find as it breaks the syntax colouring I have!